src/core/Table/hooks/index.ts

```diff
@@ -0,0 +1,6 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+export { EXPANDER_CELL_ID, useExpanderCell } from './useExpanderCell';
+export { SELECTION_CELL_ID, useSelectionCell } from './useSelectionCell';
```

Comment:
Do we need to export those ids?

Not sure if you plan to use them in your sub-rows PR or if it's just for futureproofing

Reply:
Yeah, I plan to use them. But if turns out to be a bad idea, I will remove them.